### ISSUE TITLE
Indicator View Insets

### ISSUE THREAD

**Issue by grifas** [rechsteiner/Parchment#258]
I configured my pagingViewController with 

`pagingViewController.menuItemSize = .sizeToFit(minWidth: 100, height: 40) `

and sometimes I need to have just one page. In this case there is a weird left/right offset/inset on indicator view. I also tried with your example project as you can see:

![simulator screen shot - iphone 6 - 2018-10-08 at 16 24 49](https://user-images.githubusercontent.com/9267969/46615071-ace25300-cb17-11e8-9795-2792864529f2.png)

Is it normal ?

**Comment by rechsteiner** [comment]
Hi @grifas! It seems to be a bug where the indicator insets are not being applied to both sides if you only show a single item. Thanks for catching that! I'll get that fixed shortly 🙂

If you want to get rid of the insets altogether you can set the `indicatorOptions` like this:

```
pagingViewController.indicatorOptions = .visible(
    height: 4,
    zIndex: .max,
    spacing: .zero,
    insets: .zero
)
```

**Comment by rechsteiner** [comment]
Fixed in #307. Will be included in the next version 🙂
